### PR TITLE
Forgot password

### PR DESCRIPTION
**Pull request by R-Bear**
# Design Template Forgot Password

## Task
- [x] display flex
- [x] responsive

## views
- password_reset/new.html.erb
- password_reset/edit.html.erb
### style sheet
 - password_resets.scss
## controllers
- password_resets_controller.rb
`class PasswordResetsController < ApplicationController`
  `def new`
  `end`

`def edit`
  `end`
## routes
`get 'password_resets/new'`
`get 'password_resets/edit'`

### CODE REVIEW

File: pj_movie/config/routes.rb [hunk 1]
```diff
@@ -1,3 +1,11 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+
+  get 'password_resets/new'
+  get 'password_resets/edit'
+  get 'sessions/new'
```
Comment (N310) [inline pj_movie/config/routes.rb]: you should remove this line.

File: pj_movie/app/helpers/sessions_helper.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+module SessionsHelper
```
Comment (N310) [inline pj_movie/app/helpers/sessions_helper.rb]: you should remove this file.

File: pj_movie/app/views/password_resets/new.html.erb [hunk 3]
```diff
@@ -0,0 +1,13 @@
+
+
```
Comment (N310) [inline pj_movie/app/views/password_resets/new.html.erb]: you should remove unneeded empty line.

File: pj_movie/app/views/password_resets/edit.html.erb [hunk 4]
```diff
@@ -0,0 +1,14 @@
+
```
Comment (N310) [inline pj_movie/app/views/password_resets/edit.html.erb]: you should remove unneeded empty line.

File: pj_movie/config/routes.rb [hunk 5]
```diff
@@ -1,3 +1,11 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+
+  get 'password_resets/new'
+  get 'password_resets/edit'
```
Comment (N310) [inline pj_movie/config/routes.rb]: you should use `resources`.

File: pj_movie/app/assets/stylesheets/password_resets.scss [hunk 6]
```diff
@@ -0,0 +1,56 @@
+// Place all the styles related to the PasswordResets controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
```
Comment (N310) [inline pj_movie/app/assets/stylesheets/password_resets.scss]: you should remove unneeded comment.

File: pj_movie/app/assets/stylesheets/password_resets.scss [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+html{
```
Comment (N310) [inline pj_movie/app/assets/stylesheets/password_resets.scss]: you have to move common style to other css file .
you can write style of related to password rest in this file.

**Comment by R-Bear** [comment]
I edited 
   - remove line free
   - destroyed controller Sessions
   - routes: `resource :password_resets, only: [:new, :edit]`
for this file `password_resets.coffee` I think I will use next time. Can I keep it?

File: pj_movie/app/assets/stylesheets/password_resets.scss [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+          border-radius: 5px;
+        }
+        .btn {
+        height: 45px;
+        text-align: center;
+        width: 374px;
+        background-color: #4267B2;
+        border-radius: 5px;
+        font-size: 14px;
+        margin-bottom: 40px;
```
Comment (N310) [inline pj_movie/app/assets/stylesheets/password_resets.scss]: I suggest that you add indent.

**Comment by N310** [comment]
@R-Bear 
You should resolve conflicting file.

File: pj_movie/config/routes.rb [hunk 9]
```diff
@@ -2,8 +2,11 @@
...
   get 'admin', to: 'admin/top_page#index', as: :admin
 
+  resources :static_pages, only: [:index]
+  resource :password_resets, only: [:new, :edit]
+
   namespace :admin do
     resources :top_page, only: [:index]
+    resource :password_resets, only: [:new, :edit]
   end
-  resources :static_pages, only: [:index]
```
Comment (PechBunsinh) [inline pj_movie/config/routes.rb]: you should remove this line if you have already in line 5

File: pj_movie/app/views/admin/password_resets/edit.html.erb [hunk 10]
```diff
@@ -0,0 +1,36 @@
...
+  <head>
+    <title><%= yield(:title) %> | Project Movie</title>
+    <%= csrf_meta_tags %>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <%= stylesheet_link_tag    'application', media: 'all',
+                                          'data-turbolinks-track': 'reload' %>
+    <%= stylesheet_link_tag "reset" %>
+    <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
+  </head>
+  <body>
```
Comment (N310) [inline pj_movie/app/views/admin/password_resets/edit.html.erb]: you should remove this lines.

File: pj_movie/app/views/admin/password_resets/edit.html.erb [hunk 11]
```diff
@@ -0,0 +1,36 @@
...
+                <%= f.submit "Submit", class: 'btn' %>
+                <% end %>
+              </form>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </body>
+</html>
```
Comment (N310) [inline pj_movie/app/views/admin/password_resets/edit.html.erb]: you should remove this lines.

File: pj_movie/app/views/admin/password_resets/new.html.erb [hunk 12]
```diff
@@ -0,0 +1,34 @@
...
+  <head>
+    <title><%= yield(:title) %> | Project Movie</title>
+    <%= csrf_meta_tags %>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <%= stylesheet_link_tag    'application', media: 'all',
+                                          'data-turbolinks-track': 'reload' %>
+    <%= stylesheet_link_tag "reset" %>
+    <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
+  </head>
+  <body>
```
Comment (N310) [inline pj_movie/app/views/admin/password_resets/new.html.erb]: you should remove this lines.

File: pj_movie/app/views/admin/password_resets/new.html.erb [hunk 13]
```diff
@@ -0,0 +1,34 @@
...
+                  <%= f.submit "Submit", class: 'btn' %>
+                <% end %>
+              </form>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </body>
+</html>
```
Comment (N310) [inline pj_movie/app/views/admin/password_resets/new.html.erb]: you should remove this lines.

File: pj_movie/app/views/admin/password_resets/edit.html.erb [hunk 14]
```diff
@@ -0,0 +1,22 @@
+<div class="container_admin">
```
Comment (loemhuolay) [inline pj_movie/app/views/admin/password_resets/edit.html.erb]: you should use more specific name than this
Ex:  <div class="container_admin"> to <div class="container_password_reset_admin">

**Pull request merged**